chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/developer/attestation/DeveloperAttestationCheckInReportDataCollector.java

```diff
@@ -154,10 +154,13 @@ private List<Long> getDeveloperIdsFromDeveloperAttestationReport() {
 
     private String getAttestationResponse(ChangeRequest cr, Long attestationConditionId) {
        ChangeRequestAttestationSubmission details = (ChangeRequestAttestationSubmission) cr.getDetails();
+       /*
```

Comment:
needed?

Reply:
This is code that is "broken" until we fix the reports in ticket OCD-3980.  Left it in to mark where we need to start, when fixing the reports.